features.md

```diff
@@ -160,3 +160,78 @@ Known limitations
 > editors, e.g. VSCode, provide a way to preview the rename changes before
 > applying them; while some just apply the changes directly.
 {:.tip}
+
+## Semantic Highlighting
+
+clangd computes semantic highlight tokens to enhance the source code theming. {.v9}
+
+![screenshot: highlight](screenshots/semantic_highlight.png)
+
+Tokens may be associated with one highlight kind and multiple highlight modifiers, as defined by the following section.
+
+### Kinds
+
+clangd supports following standard LSP token kinds:
+
+| Kind                   | LSP Name      | Note                       
+|------------------------|---------------|--------------------------
+| Variable{.v9}          | variable      |                          
```

Comment:
Unfortunately, the markdown implementation we're using does not support specifying a class on a table cell (https://github.com/gettalong/kramdown/issues/417).

Instead, maybe we can add a column called "Since Version" and just put the version there in plain text?